src/python/grpcio/grpc/_cython/_cygrpc/channel.pyx.pxi

```diff
@@ -430,9 +431,14 @@ cdef class Channel:
 
   def next_call_event(self):
     def on_success(tag):
-      _process_integrated_call_tag(self._state, tag)
+      if tag is not None:
+        _process_integrated_call_tag(self._state, tag)
+    if is_fork_support_enabled():
+      queue_deadline = time.time() + 1.0
```

Comment:
Also here.  I'm really worried about adding a deadline, its not clear that this will have the correct behavior.